test/component/CustomsDeclarationClearanceSpec.scala

```diff
@@ -105,100 +103,15 @@ class CustomsDeclarationClearanceSpec extends ComponentTestSpec with AuditServic
     }
 
-    scenario("An unauthorised CSP is not allowed to submit a customs clearance declaration") {
-      Given("A CSP wants to submit a valid customs clearance declaration")
-      val request: FakeRequest[AnyContentAsXml] = ValidClearanceRequest.fromCsp.postTo(endpoint)
-
-      And("the CSP is unauthorised with its privileged application")
-      authServiceUnauthorisesScopeForCSP()
-      authServiceUnauthorisesCustomsEnrolmentForNonCSP(cspBearerToken)
-
-      When("a POST request with data is sent to the API")
-      val result: Future[Result] = route(app = app, request).value
-
-      Then("a response with a 401 (UNAUTHORIZED) status is received")
-      status(result) shouldBe UNAUTHORIZED
-
-      And("the response body is Unauthorised error")
-      string2xml(contentAsString(result)) shouldBe string2xml(UnauthorisedRequestError)
-
-      And("the request was authorised with AuthService")
-      verifyAuthServiceCalledForCsp()
-    }
-
-    scenario("A non-CSP successfully submits a declaration on behalf of somebody with Customs enrolment") {
-      Given("A Software House wants to submit a valid customs clearance declaration")
-
-      val request: FakeRequest[AnyContentAsXml] = ValidClearanceRequest.fromNonCsp.postTo(endpoint)
-      startApiSubscriptionFieldsService(apiSubscriptionKeyForXClientIdV2)
-
-      And("declarant is enrolled with Customs having an EORI number")
-      authServiceUnauthorisesScopeForCSP(nonCspBearerToken)
-      authServiceAuthorizesNonCspWithEori()
-
-      When("a POST request with data is sent to the API")
-      val result: Future[Result] = route(app = app, request).value
-
-      Then("a response with a 202 (ACCEPTED) status is received")
-      status(result) shouldBe ACCEPTED
-
-      And("the response body is empty")
-      contentAsString(result) shouldBe 'empty
-
-      And("the request was authorised with AuthService")
-      verifyAuthServiceCalledForNonCsp()
-    }
-
-    scenario("A non-CSP is not authorised to submit a declaration on behalf of somebody without Customs enrolment") {
-      Given("A Software House wants to submit a valid customs clearance declaration")
-
-      val request: FakeRequest[AnyContentAsXml] = ValidClearanceRequest.fromNonCsp.postTo(endpoint)
-
-      And("declarant is not enrolled with Customs")
-      authServiceUnauthorisesScopeForCSP(nonCspBearerToken)
-      authServiceUnauthorisesCustomsEnrolmentForNonCSP()
-
-      When("a POST request with data is sent to the API")
-      val result: Future[Result] = route(app = app, request).value
-
-      Then("a response with a 401 (UNAUTHORIZED) status is received")
-      status(result) shouldBe UNAUTHORIZED
-
-      And("the response body is empty")
-      string2xml(contentAsString(result)) shouldBe string2xml(UnauthorisedRequestError)
-
-      And("the request was authorised with AuthService")
-      verifyAuthServiceCalledForNonCsp()
-    }
   }
 
   //*** UNHAPPY PATH SPECS ***
```

Comment:
comments, do we need them 
also multiple features with one scenario in each one of them, should we merge them?

Reply:
No, I think it makes it clear what is being tested and separates them nicely i.e.
- "Submissions with v1.0 accept header"
- "Declaration API authorises submissions from CSPs and Software Houses with v2.0 accept header"
- "The API handles errors as expected" give

Reply:
Will remove UNHAPPY comments

Reply:
scenarios are being used for separation.

Reply:
actually, it might be better if we take a view from QAs on this